examples/dlint/color.rs

```diff
@@ -1,4 +1,5 @@
 // Copyright 2018-2024 the Deno authors. All rights reserved. MIT license.
+
```

Comment:
This space is necessary to prevent auto-imports being inserted above the copyright statement.

Reply:
Nice, I've hit it so many times...